viewing_party/party.py

```diff
@@ -1,23 +1,149 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        return None

```

Comment:
Great validation! One small thing, we should use `is None` instead of `== None`.